src/Fable.Transforms/Replacements.fs

```diff
@@ -3394,6 +3395,14 @@ let tryBaseConstructor com ctx (ent: Entity) (argTypes: Lazy<Type list>) genArgs
             | _ -> failwith "Unexpected stack constructor"
         let entityName = FSharp2Fable.Helpers.cleanNameAsJsIdentifier "Stack"
         Some(makeImportLib com Any entityName "Stack", args)
+    | Types.queue ->
+        let args =
+            match argTypes.Value, args with
+            | [], _ ->
+                []
+            | _ -> failwith "Unexpected queue constructor"
```

Comment:
I just realized that Stack is doing the same, but there are actually other overloads for the constructor so probably we shouldn't do a hard crash here.

Reply:
I was puzzled by this code, but had just copied it from the stack code. I felt it should be related to the 3 constructors. Shall I take a pass at reworking that in a separate commit?